src/asterPlotSettingsModel.ts

```diff
@@ -211,11 +235,61 @@ class LabelsCardSettings extends BaseFontCardSettings {
             maxValue: { value: 17, type: ValidatorType.Max },
         }
     });
+     name: string = "values";
+    displayName: string = "Values";
+    displayNameKey: string = "Visual_Values";
+    slices = [this.displayUnits, this.precision, this.font, this.color];
+}
+
+class LabelsCardSetting extends formattingSettings.CompositeCard {
+    show = new formattingSettings.ToggleSwitch({
+        name: "show",
+        displayName: "Show",
+        displayNameKey: "Visual_Show",
+        value: false,
+    });
+
+    topLevelSlice = this.show;
+
+    public labelsOptionsGroup: LabelsOptionsSettingsGroup = new LabelsOptionsSettingsGroup();
+    public labelsValuesGroup: LablesValuesSettingsGroup = new LablesValuesSettingsGroup();
 
     name: string = AsterPlotObjectNames.Labels.name;
     displayName: string = AsterPlotObjectNames.Labels.displayName;
     displayNameKey: string = AsterPlotObjectNames.Labels.displayNameKey;
-    slices = [this.displayUnits, this.precision, this.font, this.color];
+    groups: formattingSettings.Group[] = [this.labelsOptionsGroup, this.labelsValuesGroup];
+
+    get color() {
```

Comment:
defining clear properties or getters for this class might be a good idea. But you can avoid it here and use `...labels.labelsValuesGroup.color.value.value` in the code where you want to get this value instead of `...labels.color.value.value`, since you already defined `labelsValuesGroup` property.